resolwe_bio/tests/unit/test_management_commands.py

```diff
@@ -0,0 +1,55 @@
+# pylint: disable=missing-docstring
+from django.core.management import call_command
+
+from resolwe.flow.models import Data, Storage
+from resolwe_bio.utils.test import BioProcessTestCase
+from resolwe_bio.models import Sample
+
+
+class GenerateSamplesTest(BioProcessTestCase):
+
+    def test_generate_samples(self):
+        call_command('generate_samples', '-s=1', '-p=0', '--rseed')
+        sample = Sample.objects.last()
+        if sample:
+            sample_data = sample.data.all().order_by('id')
```

Comment:
If there is no `ordering` specified in the model's `Meta`, then the ordering is undefined and may be arbitrary. If you need a specific order, you must always specify `order_by` (or set the default ordering in model metadata).